x-pack/plugins/infra/server/routes/log_analysis/results/log_entry_rate_examples.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import Boom from 'boom';
+import { fold } from 'fp-ts/lib/Either';
+import { identity } from 'fp-ts/lib/function';
+import { pipe } from 'fp-ts/lib/pipeable';
+
+import { throwErrors } from '../../../../common/runtime_types';
+import { InfraBackendLibs } from '../../../lib/infra_types';
+import { NoLogAnalysisResultsIndexError, getLogEntryRateExamples } from '../../../lib/log_analysis';
+import { assertHasInfraMlPlugins } from '../../../utils/request_context';
+import {
+  getLogEntryRateExamplesRequestPayloadRT,
+  getLogEntryRateExamplesSuccessReponsePayloadRT,
+  LOG_ANALYSIS_GET_LOG_ENTRY_RATE_EXAMPLES_PATH,
+} from '../../../../common/http_api/log_analysis';
+
+const anyObject = schema.object({}, { unknowns: 'allow' });
+
+export const initGetLogEntryRateExamplesRoute = ({ framework, sources }: InfraBackendLibs) => {
+  framework.registerRoute(
+    {
+      method: 'post',
+      path: LOG_ANALYSIS_GET_LOG_ENTRY_RATE_EXAMPLES_PATH,
+      validate: {
+        // short-circuit forced @kbn/config-schema validation so we can do io-ts validation
+        body: anyObject,
```

Comment:
How about directly validating this using

```suggestion
        body: createValidationFunction(getLogEntryRateExamplesRequestPayloadRT),
```

?